peers.yaml

```diff
@@ -939,3 +939,13 @@ AS6695:
     description: DE-CIX Route servers
     import: AS-DECIX AS-DECIX-V6
     export: AS8283:AS-COLOCLUE
+
+AS51706:
+    description: France-IX Route servers
```

Comment:
nit - You may want to consider using France-IX (Paris) here - Marseille has a different RS.

Same goes above for line 939 for DE-CIX (Frankfurt)